library/smarty/plugins/function.headerShow.php

```diff
@@ -23,6 +23,4 @@
  * @param Smarty
  */
 function smarty_function_headerShow($params, &$smarty)
```

Comment:
remove this entire script and then remove the following string from the codebase (about 18 of them):
```
{headerShow}
```